apps/davi/src/Modules/Guilds/pages/CreateProposal.tsx

```diff
@@ -83,16 +84,23 @@ const CreateProposalPage: React.FC = () => {
   ]);
   const [isPermissionWarningModalOpen, setIsPermissionWarningModalOpen] =
     useState(false);
+
+  const [html, onHTMLChange] = useLocalStorageWithExpiry<string>(
+    `${guildId}/create-proposal/html`,

```

Comment:
Maybe the `create-proposal` key could be replaced with the `discussionId` here? What do you think?

Reply:
Yeah. It will provide support for maintaining different `GuildID-DiscussionID` draft combinations.